pyasic/miners/backends/braiins_os.py

```diff
@@ -720,6 +722,10 @@ async def upgrade_firmware(self, file: Path):
             "_get_uptime",
             [RPCAPICommand("rpc_summary", "summary")],
         ),
+        str(DataOptions.POOLS): DataFunction(
+            "_get_pools",
+            [RPCAPICommand("grpc_pools", "pools")]
```

Comment:
BOSer uses gRPC as its web API, so you would use `WebAPICommand` here.

Reply:
This will also need to have the `"pools"` command name changed to `"get_pool_groups"`, and probably have the `"grpc_pools"` changed to `"grpc_pool_groups"` to be consistent with the other kwargs.

Reply:
I will make the required changes, As soon as possible. Thank you for the feedback.